components/ui/Feed/FeedView.tsx

```diff
@@ -238,27 +152,6 @@ function FeedView({ feed, community = false, header }: FeedViewProps) {
     />
   );
 
-  const footer = () => {
-    if ((feed.postsLoading && feed.posts.length > 0) || endReached) {
-      return <LoadingFooter message="Loading more posts..." />;
-    }
-    if (feed.postsError) {
-      if (!feed.posts || feed.posts.length < 1) {
-        return <LoadingErrorView onRetryPress={feed.doLoad} />;
-      }
-
-      return (
-        <LoadingErrorFooter
-          onRetryPress={() => feed.doLoad(true)}
-          message="Failed to load posts :("
-        />
-      );
-    }
-    return null;
-  };
-
-  const HeaderComponent = header;
```

Comment:
This seemed unnecessary, and is in fact kinda weird given `header` is completely optional and can be `undefined`. But i could be wrong. Either way, i think it's more straight forward to just render it out via `{header}` instead of abstracting it into the `HeaderComponent`.